.gitignore

```diff
@@ -4,3 +4,4 @@ vendor
 composer.lock
 
 nbproject
+.idea
```

Comment:
@looking-4-justice Can you please implement this `.idea` change to all of our main codebases? I will review them. Thank you! ^^

list:
* content-hub-d8 (1.x and 2.x)
* content-hub-d7
* content-hub-php (1.x and 2.x)
* content-hub-ember (1.x and 2.x)
* hive

Please and thank you! ^^

Reply:
@nkarpenko Can you please implement this `.idea` change to all of our main plexus codebases? I will review them.

list:
* plexus
* plexus-client-go
* customer-auth
* (any other codebases)

Please and thank you! ^^

Reply:
By all means. Just a couple of questions:
- Do I need a ticket for this (hence a branch with the same number)?
- Which branch should I branch off of for each project?

Reply:
Documenting as discussed:
* No because it is anticipated to be less than 30 mins work.
* Master or develop or equivalent.